src/Servers/IIS/build/testsite.props

```diff
@@ -55,33 +55,30 @@
     <PropertyGroup>
       <InjectDepsAssembly>$(MSBuildThisFileDirectory)..\test\testassets\TestTasks\bin\$(Configuration)\$(TargetFramework)\TestTasks</InjectDepsAssembly>
       <InjectDepsApp Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' ">$(InjectDepsAssembly)</InjectDepsApp>
-      <InjectDepsArguments>"win7-$(NativePlatform)" "$(AncmInProcessRHPath)"</InjectDepsArguments>
     </PropertyGroup>
 
     <PropertyGroup Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' ">
-      <InjectDepsAssembly>$(InjectDepsAssembly).exe</InjectDepsAssembly>
-      <InjectDepsApp>$(InjectDepsAssembly)</InjectDepsApp>
+      <InjectDepsApp>$(InjectDepsAssembly).exe</InjectDepsApp>
```

Comment:
Does this ever get used? .deps.json is not a .NET Framework concept.

Reply:
I'm confused, help me understand. The property group says `Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' ">`. Doesn't this file get imported by the test site project? So  `TargetFrameworkIdentifier` comes from the test site project?

Reply:
Yeah, you are correct. Deleting all of the .NET Framework related things.